CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 * **Removed**
    * REMOVED: Docker image pushes to Dockerhub [#4033](https://github.com/valhalla/valhalla/pull/4033)
 * **Bug Fix**
+   * FIXED: docker image fails building for arm64 [#3950](https://github.com/valhalla/valhalla/issues/3950)
```

Comment:
we organize our changelog in a date-descending manner, i.e. newest comes last. also we refer to the pull request, not the issue.

Reply:
np;)